cloud/huawei/pods.go

```diff
@@ -0,0 +1,137 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package huawei
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"os/exec"
+	"text/template"
+	"time"
+
+	"github.com/golang/glog"
+	"sigs.k8s.io/cluster-api/cloud/huawei/config"
+)
+
+var apiServerImage = "gcr.io/k8s-cluster-api/cluster-apiserver:0.0.2"
+var controllerManagerImage = "gcr.io/k8s-cluster-api/controller-manager:0.0.2"
+var machineControllerImage = "gcr.io/karangoel-gke-1/huawei-machine-controller:0.0.1-dev"
```

Comment:
You shouldn't be able to publish to my project :)

Reply:
:sweat_smile: